src/reduced_basis/rb_construction.C

```diff
@@ -1303,11 +1303,12 @@ void RBConstruction::enrich_basis_from_rhs_terms(const bool resize_rb_eval_data)
               rhs->print_matlab(std::string(temp_file));
             }
         }
+#endif // 0
 
       // Debugging: enable this code to print the most recent truth
       // solution to a uniquely-named file.
 #ifdef LIBMESH_HAVE_EXODUS_API
-      if (false)
```

Comment:
So clang warns about `if (false)` now? I find that kind of annoying because with `#if 0` it's too easy to introduce typos and other errors while refactoring, as the code is no longer seen by the compiler at all. I think with `if (false)` it at least still reports errors.

Reply:
That's a good point.  That's more than just a little annoying.  I don't see any other way around it, though, unless we revoke clang's -Wunreachable-code privileges in compiler.m4.  That might be worth considering, since we'll be getting roughly the same warning coverage in our gcc builds?  But it might just be a matter of time before gcc adopts the same interpretation; it's hard to argue that the stuff inside ```if (false)``` isn't indeed unreachable code.